feat: object store to pvc migrations (#633)
replicatedhq/kurl.sh: src/markdown-pages/add-ons/velero.md

### Configure Backend Object Store

> Velero requires a backend object store where it will save your backups.

Reviewer: This isn't totally accurate anymore since we're supporting a PV now, right?

For the initial install, a storage location inside the cluster will be used as the default in this order of precedence:
1. The `kotsadm.disableS3` flag is set to `true` in the installer spec. A Persistent Volume (PV) will be used as the storage backend.

> 1. There is an object storage provider in the cluster. This could be satisfied by the Rook or Minio addons.

Reviewer 2: ```suggestion
1. There is an object storage provider in the cluster. This could be satisfied by the Rook or Minio add-ons.
```

1. Given none of these conditions, no default storage location will be configured.

> You can also decide to move between these "Internal" storage locations. See [Removing Minio](/docs/install-with-kurl/removing-minio) for more information on this migration.

Reviewer 2: ```suggestion
You can also decide to move between these **Internal** storage locations. See [Removing Minio](/docs/install-with-kurl/removing-minio) for more information on this migration.
```

> None of these "Internal" locations are suitable for disaster recovery because loss of the cluster will mean loss of backups. 

Reviewer 2: ```suggestion
**Warning:** None of these **Internal** locations are suitable for disaster recovery because loss of the cluster will mean loss of backups. 
```
